.travis.yml

```diff
@@ -13,7 +13,8 @@ script:
       flake8 . &&
       mypy . &&
       isort --check-only -rc . &&
-      black --check .
+      black --check . &&
+      docformatter --recursive --check loggo/ tests/ setup.py
```

Comment:
The reason for `docformatter --recursive --check loggo/ tests/ setup.py` instead of `docformatter --recursive --check .` is because docformatter has no way to ignore files. Using a `.` would make it try to format site-packages etc.

Reply:
Good to know :+1: